bot/code_review_bot/report/__init__.py

```diff
@@ -29,6 +29,8 @@ def get_reporters(configuration):
             if "reporter" not in conf:
                 raise Exception("Missing reporter declaration")
             name = conf["reporter"]
+            if name != "build_error":
+                continue
```

Comment:
What is this change? Maybe it should be in a separate PR?

Reply:
Remains from a local test, i'm removing it